src/router.ts

```diff
@@ -704,6 +731,11 @@ const routeVoterToSlackChannel = async (
     `timestampOfLastMessageInThread: ${timestampOfLastMessageInThread}`
   );
 
+  await DbApiUtil.setThreadNeedsAttentionToDb(
```

Comment:
This updates the thread to which a voter is returning to have the needsAttention status of the thread from which they are coming from, right? Can you add a comment to that effect?